packages/signature-controller/src/SignatureController.ts

```diff
@@ -565,6 +596,40 @@ export class SignatureController extends BaseControllerV2<
     );
   }
 
+  tryForEachMessageManager(
+    callbackFn: (
+      messageManager: AbstractMessageManager<any, any, any>,
+      ...args: any[]
+    ) => boolean,
+    ...args: any[]
+  ) {
+    const messageManagers = [
+      this.#messageManager,
+      this.#personalMessageManager,
+      this.#typedMessageManager,
+    ];
+
+    for (const manager of messageManagers) {
+      if (callbackFn(manager, ...args)) {
+        return true;
+      }
+    }
+    return false;
```

Comment:
Rather than returning a boolean that I don't think is used, shall we instead throw an error if the message can't be found in any of the message managers?

Something like: `Message not found for id: ${messageId}`?